src/bindings/python/src/openvino/utils.py

```diff
@@ -38,6 +39,25 @@ def _add_openvino_libs_to_search_path() -> None:
                 os.add_dll_directory(os.path.abspath(lib_path))
 
 
+def get_cmake_path() -> str:
+    """Searches for the directory containing CMake files within the package install directory.
+
+    Parameters:
+    None
+
+    Returns:
+    str: The path to the directory containing CMake files, if found. Otherwise, returns empty string.
```

Comment:
```suggestion

    :return: The path to the directory containing CMake files, if found. Otherwise, returns empty string.
    :rtype: str
```
usually we specify docstrings like this